tests/protocol/ipsec/ike-zero-length.zeek

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2021 by the Zeek Project. See LICENSE for details.
+
```

Comment:
The schema for `weird.log` has changed between zeek3 and zeek4, see [this comment](https://github.com/zeek/spicy-analyzers/pull/100#discussion_r751194679).